manifest.json

```diff
@@ -15,10 +15,10 @@
   },
   "externally_connectable": {
     "matches": [
-      "http://*/*",
-      "https://*/*"
+      "*://*.b12.io/*"
```

Comment:
The previous wildcards are invalid—let's commit this as an example for others? Don't think there's any harm since we're not planning to be malicious 😎